classes/pdf/HTMLTemplateInvoice.php

```diff
@@ -29,13 +29,20 @@
  */
 class HTMLTemplateInvoiceCore extends HTMLTemplate
 {
+    /**
+     * @var Order
+     */
     public $order;
+
+    /**
+     * @var OrderInvoice
+     */
     public $order_invoice;
-    public $available_in_your_account = false;
```

Comment:
Although it seems to be useless, we cannot remove it, it would be a breaking change if someone extended this class 😢

Reply:
This property is in parent HTMLTemplate and moved assignation in constructor, so no breaking change for existing override